### DISCUSSION TITLE
Support `--archive` flag

### DISCUSSION THREAD

**Discussion by pm0u** [BetaHuhn/deploy-to-vercel-action#391] [Ideas]
Vercel CLI allows an `--archive` flag which can compress the upload into a single tgz archive before upload. This can greatly reduce the number of files uploaded and reduce impact on API limits (5,000/day free, 40,000/day paid plan).

https://vercel.com/docs/cli/deploy#archive

**Comment by mountainash** [comment]
@pm0u this was one of the very first features added to https://github.com/mountainash/deploy-to-vercel-action
